core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -351,20 +351,31 @@ angular.module('oppia').component('contributionsAndReview', {
         ctrl.dropdownShown = !ctrl.dropdownShown;
       };
 
-      ctrl.loadContributions = function() {
+      ctrl.loadOpportunities = function() {
+        return ctrl.loadContributions(/* Param shouldResetOffset= */ true);
```

Comment:
Do we do this `/* ... */` style elsewhere in the codebase?

(I like the idea of having it optionally in cases where it's not clear what the arg means, not sure if it's worth generalizing. /cc @U8NWXD @srijanreddy98 @vojtechjelinek )

Reply:
Ah, hm, I think the way you have it currently is fine IMO.